avocado/core/runners/avocado_instrumented.py

```diff
@@ -45,28 +95,15 @@ def _run_avocado(runnable, queue):
                          'run.results_dir': tempfile.mkdtemp(),
                          }]
 
+        AvocadoInstrumentedTestRunner._start_logging(runnable, queue)
         instance = loader.loader.load_test(test_factory)
         early_state = instance.get_state()
+        early_state['type'] = "early_state"
         queue.put(early_state)
         instance.run_avocado()
         state = instance.get_state()
-        # This should probably be done in a translator
-        if 'status' in state:
-            status = state['status'].lower()
-            final_status = [_.lower() for _ in teststatus.user_facing_status]
-            if status in final_status:
-                state['result'] = status
-                state['status'] = 'finished'
-            else:
-                state['status'] = 'running'
-
-        # This is a hack because the name is a TestID instance that can not
-        # at this point be converted to JSON
-        if 'name' in state:
-            del state['name']
-        if 'time_start' in state:
-            del state['time_start']
-        queue.put(state)
+        _send_message(state['whiteboard'], queue, 'whiteboard')
```

Comment:
At some point in the past, discussion with @clebergnu, I had the impression that we are not using 'whiteboard' anymore.

@clebergnu do we still need this?

Reply:
This the firs time I am hearing about it. And actually I don't see a reason to remove this feature from avocado-instrumented-tests.